package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "mapswipe",
-  "version": "2.1.9",
-  "build": "5",
+  "version": "2.2.1",
+  "build": "4",
```

Comment:
Any reason we decrease the build? I think we already had a discussion on this but I am a bit hazy on the details